sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -11,15 +11,10 @@
 import collections
 from typing import Any, Dict, Tuple, List, Optional, TYPE_CHECKING, cast, Union
 from datetime import timedelta
-
-try:
-    from urlparse import urlparse
-    from urllib import quote_plus  # type: ignore
-except ImportError:
-    from urllib.parse import urlparse, quote_plus
-
-from uamqp import AMQPClient, Message, authentication, constants, errors, compat, utils
+from urllib.parse import urlparse, quote_plus
 import six
```

Comment:
We can also get rid of six here since its being used in one place below. Since for py3+ six.binary_type will always be bytes.
https://six.readthedocs.io/#six.binary_type